packages/shared-block-ring/shared-block-ring.3.0.0/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+synopsis: "A single-consumer single-producer queue on a block device"
+description: """
+This is a simple queue containing variable-length items stored on a
+          disk, in the style of Xen shared-memory-ring."""
+maintainer: "jonathan.ludlam@citrix.com"
+authors: ["David Scott" "Jon Ludlam" "Si Beaumont" "Pau Ruiz Safont"]
+license: "ISC"
+tags: ["org:mirage" "org:xapi-project"]
+homepage: "https://github.com/mirage/shared-block-ring"
+bug-reports: "https://github.com/mirage/shared-block-ring/issues/"
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "bisect_ppx" {dev & >= "2.5.0"}
+  "cmdliner"
+  "cstruct" {>= "3.0.0"}
+  "dune" {>= "2.7.0"}
+  "duration"
+  "io-page"
+  "io-page-unix" {>= "2.0.0"}
+  "logs"
+  "lwt"
+  "lwt_log"
+  "mirage-block" {>= "2.0.1"}
+  "mirage-block-unix"
+  "mirage-clock" {>= "3.0.0"}
+  "mirage-clock-unix" {with-test}
+  "mirage-time" {>= "2.0.1"}
+  "mirage-time-unix"
+  "ounit2" {with-test}
+  "ppx_cstruct"
+  "ppx_sexp_conv" {>= "v0.10.0"}
```

Comment:
```suggestion
  "ppx_sexp_conv" {>= "v0.12.0"}
```
`ppx_sexp_conv` seems to only support dune since v0.12. See https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/e1e996b08bff8ab41bde0b19fcaf106299a672f3/variant/opam-2.1,compilers,4.06,shared-block-ring.3.0.0,lower-bounds for details